.github/workflows/build-test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           profile: minimal
```

Comment:
Not sure if its worth it, but we may have a separated workflow for risc0 if needed. Wdyt @bacv ?